pkg/operator/ceph/csi/controller.go

```diff
@@ -49,6 +50,14 @@ type ReconcileCSI struct {
 	context          *clusterd.Context
 	opManagerContext context.Context
 	opConfig         opcontroller.OperatorConfig
+	multusClusters   []ClustersInfo
+}
+
+// ClustersInfo is a struct that holds the information of a cluster, it knows its internals (like
+// FSID through clusterInfo) and the Kubernetes object that represents it (like the CephCluster CRD)
+type ClustersInfo struct {
```

Comment:
What about just calling it `cluster` or perhaps `clusterDetail`? ClustersInfo sounds plural, but it's not, and looks like it's not used outside this package.
```suggestion
type cluster struct {
```